### DISCUSSION TITLE
Block by default

### DISCUSSION THREAD

**Discussion by CygnusBl4ck** [tnodir/fort#319] [General]
Hi!

First of all, thanks for creating and maintaining Fort ❤️
I'm sorry if the following question already got answered before but I'd like to know how to configure Fort to prompt the Block/Allow message when a new application wants to connect to the internet, blocking it by default until the question is answered.
I guess it's configured to allow connections natively.

Thanks in advance.

**Comment by tnodir** [comment]
Hi. Set "Filter mode" to "Block, if not allowed".
